pkgs/ffigen/test/large_integration_tests/_expected_libclang_bindings.dart

```diff
@@ -8321,6 +9193,20 @@ enum CXSymbolRole {
 
   final int value;
   const CXSymbolRole(this.value);
+
+  static CXSymbolRole fromValue(int value) => switch (value) {
+        0 => CXSymbolRole_None,
+        1 => CXSymbolRole_Declaration,
+        2 => CXSymbolRole_Definition,
+        4 => CXSymbolRole_Reference,
+        8 => CXSymbolRole_Read,
+        16 => CXSymbolRole_Write,
+        32 => CXSymbolRole_Call,
+        64 => CXSymbolRole_Dynamic,
+        128 => CXSymbolRole_AddressOf,
+        256 => CXSymbolRole_Implicit,
```

Comment:
This suspiciously look like flags that have to be combined.

Hopefully no one uses enums flags in their C APIs, because then we cannot pass `1 & 2` from Dart anymore, as `3` is not a valid enum value. 😄

Reply:
Oh, that's an interesting point. Is that an issue? It would only be an issue if the function itself accepts an enum, because if it accepted any int we can use `CXSymbolRole_Call.value`. But even in C, `3` wouldn't be a valid enum value anyway, so I would assume this isn't an actual issue.

Reply:
Yeah that's what I assume too. But C probably allows you to bitwise combine enums as ints and pass in an int into an enum. 🙈 

If someone runs into this, we'll find out soon enough I assume. 🙃